emails/templates/emails/wrapped_email.html

```diff
@@ -146,6 +147,13 @@
 
         </td>
         <td class="header-block-right" style="vertical-align: bottom;" width="50%" align="right">
+          {% flag 'holiday_promo_2023' %}
+          {% if not has_premium %}
+          <span style="display: block; filter: grayscale(100%);">
```

Comment:
Why `filter: grayscale(100%);`? Did you intend to drain the color from the gift characters?

![Screenshot 2023-12-08 at 11 10 37 AM](https://github.com/mozilla/fx-private-relay/assets/286017/affe05fb-e0a7-48b8-b880-9a558c9ccb09)

Without it:

![Screenshot 2023-12-08 at 11 11 34 AM](https://github.com/mozilla/fx-private-relay/assets/286017/d941f539-4a03-4207-854b-fe7cd8a69212)

Reply:
Yes. In [the figma](https://www.figma.com/file/nUvE6f4BWV5mugnynEJFnx/Relay-Holiday-promo-Banner?type=design&node-id=271-1175&mode=design&t=rfdFMig4GRr7LsJr-0), th emoji was grayscale to match rest of the header.